tests/BaseTest.php

```diff
@@ -2,9 +2,10 @@
 
 namespace PHPChunkit\Test;
 
-use PHPUnit_Framework_TestCase;
+require_once __DIR__ . '/bootstrap.php';
+use PHPUnit\Framework\TestCase;
```

Comment:
Can you add a new line between the require_once and the use statement?

Reply:
Done.